sources/PTYSession.m

```diff
@@ -3503,10 +3491,8 @@ - (void)scheduleUpdateIn:(NSTimeInterval)timeout {
 
 - (void)doAntiIdle {
     NSTimeInterval now = [NSDate timeIntervalSinceReferenceDate];
-    if (now >= _lastInput + 60) {
-        [_shell writeTask:[NSData dataWithBytes:&_antiIdleCode length:1]];
-        _lastInput = now;
-    }
+    [_shell writeTask:[NSData dataWithBytes:&_antiIdleCode length:1]];
```

Comment:
Ah yes. I think I missed that in the manual copy from my own hacked-up fork over to a clean reference branch of your master.